packages/styling/src/styles/theme.ts

```diff
@@ -136,7 +145,7 @@ export function createTheme(theme: IPartialTheme, depComments: boolean = false):
     semanticColors: newSemanticColors,
     isInverted: !!theme.isInverted,
     disableGlobalClassNames: !!theme.disableGlobalClassNames,
-    typography: typography
+    typography: typography as ITypography
```

Comment:
We were discussing this casting a bit the other day. I think it's fine because we know that DefaultTypography has a full ITypography definition. And as we've seen in your tests, even if you override only part of typography, it maintains all of the other properties set by DefaultTypography.